script/ci

```diff
@@ -3,4 +3,8 @@
 export RANGE_SPEC_PATH=/tmp/range-spec
 git clone https://github.com/square/range-spec.git $RANGE_SPEC_PATH
 
+# pinning upstream spec repo
+pushd $RANGE_SPEC_PATH;
+git checkout 54769a7ebb70dc1b51af0c51317a61c1f76e6f03
```

Comment:
nice I've been meaning to do this for a while!

On hrange I used a submodule - undecided whether I actually like that yet or not https://github.com/xaviershay/hrange/pull/3